index.js

```diff
@@ -2,45 +2,50 @@ var fs = require('fs-extra');
 var path = require('path');
 var run = require('electron-installer-run');
 var zipFolder = require('zip-folder');
-var format = require('util').format;
 var series = require('async').series;
 var debug = require('debug')('electron-installer-zip');
 
 module.exports = function(opts, done) {
   opts.dir = path.resolve(opts.dir);
   opts.out = path.resolve(opts.out);
   opts.platform = opts.platform || process.platform;
+  if (path.extname(opts.out) === '.zip') {
```

Comment:
I wonder if we should accommodate `.ZIP`.

Reply:
Done 👍